primefaces-integration-tests/src/main/webapp/resources/composite/dataTableFilter.xhtml

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:cc="http://xmlns.jcp.org/jsf/composite"
+      xmlns:p="http://primefaces.org/ui">
+
+<cc:interface>
+</cc:interface>
+
+<cc:implementation>
+    <p:inputText id="#{cc.clientId}" type="search"
```

Comment:
This is invalid
Just set a id Like ID="Input"

Reply:
Thanks!  I was just following that example that user submitted but this makes more sense.  Trying again...